[2021-05-12 11:57:04,720] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task1 2021-05-12T10:39:48.267484+00:00 [queued]>
[2021-05-12 11:57:04,727] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: status_test.task1 2021-05-12T10:39:48.267484+00:00 [queued]>
[2021-05-12 11:57:04,728] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 11:57:04,728] {taskinstance.py:1069} INFO - Starting attempt 13 of 14
[2021-05-12 11:57:04,728] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 11:57:04,853] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): task1> on 2021-05-12T10:39:48.267484+00:00
[2021-05-12 11:57:04,856] {standard_task_runner.py:52} INFO - Started process 2230450 to run task
[2021-05-12 11:57:04,862] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'status_test', 'task1', '2021-05-12T10:39:48.267484+00:00', '--job-id', '119', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_status.py', '--cfg-path', '/tmp/tmp53kcu3if', '--error-file', '/tmp/tmp3g088y2y']
[2021-05-12 11:57:04,863] {standard_task_runner.py:77} INFO - Job 119: Subtask task1
[2021-05-12 11:57:05,070] {logging_mixin.py:104} INFO - Running <TaskInstance: status_test.task1 2021-05-12T10:39:48.267484+00:00 [running]> on host sdn-k8s-b2-3-3.maas
[2021-05-12 11:57:05,191] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=davidwu5858@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=status_test
AIRFLOW_CTX_TASK_ID=task1
AIRFLOW_CTX_EXECUTION_DATE=2021-05-12T10:39:48.267484+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-12T10:39:48.267484+00:00
[2021-05-12 11:57:05,191] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-12 11:57:05,192] {bash.py:158} INFO - Running command: echo im task 1 !!!!!!!!!!!!!!!!!!!! || exit 1
[2021-05-12 11:57:05,199] {bash.py:169} INFO - Output:
[2021-05-12 11:57:05,200] {bash.py:173} INFO - im task 1 !!!!!!!!!!!!!!!!!!!!
[2021-05-12 11:57:05,200] {bash.py:177} INFO - Command exited with return code 0
[2021-05-12 11:57:05,315] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=status_test, task_id=task1, execution_date=20210512T103948, start_date=20210512T115704, end_date=20210512T115705
[2021-05-12 11:57:05,561] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-12 11:57:05,593] {local_task_job.py:146} INFO - Task exited with return code 0
